src/CMakeLists.txt

```
@@ -375,6 +395,59 @@ if (ENABLE_BLACKMAGIC)
   endif()
 endif()
 
+################## OPENCV ###################
+
+find_package( OpenCV 4 )
+if (OpenCV_FOUND)
+  message("\nCOMPILING WITH OPENCV\n")
+  set(CMAKE_SWIG_FLAGS "-DUSE_OPENCV=1")
```

Nope.
```suggestion
  list(APPEND CMAKE_SWIG_FLAGS -DUSE_OPENCV=1)
```
That way you don't blow away all the _other_ SWIG flags.